packages/r/rebar3/package.yml

```diff
@@ -0,0 +1,19 @@
+name       : rebar3
+version    : 3.23.0
+release    : 1
+source     :
+    - https://github.com/erlang/rebar3/archive/refs/tags/3.23.0.tar.gz : 00646b692762ffd340560e8f16486dbda840e1546749ee5a7f58feeb77e7b516
+homepage   : https://rebar3.org
+license    : Apache-2.0
+component  : programming.tools
+summary    : Erlang build tool that makes it easy to compile and test Erlang applications and releases.
+description: |
+    Rebar3 is an Erlang tool that makes it easy to create, develop, and release Erlang libraries, applications, and systems in a repeatable manner.
+builddeps  :
+    erlang
+rundeps    :
+    erlang
+build      : |
+    escript bootstrap
+install    : |
```

Comment:
Do we want to include anything extra like shell completions or manpages? The [Fedora package](https://src.fedoraproject.org/rpms/erlang-rebar3/blob/rawhide/f/erlang-rebar3.spec#_75) installs a few other things, but the Arch AUR package only installs the shell completions.

Reply:
Thanks again for the review @EbonJaeger.

Sorry about the duff formatting of builddeps and rundeps, no idea why I did that. :facepalm: 

Good point about not needing `erlang` as separate runtime dependency for Gleam if already requiring `rebar3` with same dependency.

> Do we want to include anything extra like shell completions or manpages? The [Fedora package](https://src.fedoraproject.org/rpms/erlang-rebar3/blob/rawhide/f/erlang-rebar3.spec#_75) installs a few other things, but the Arch AUR package only installs the shell completions.

I've added the man pages and shell completions, similar to Arch AUR, although I've added fish and zsh completions as well as bash.

I had a look at the extra stuff Fedora adds, and [Nixpkgs](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/tools/build-managers/rebar3/default.nix#L55), but I'm struggling to work out how they would be installed on Solus, and for the current use case, I don't think they're needed. If in the future someone with more familiarity with Erlang wants to use rebar3 from Erlang as a library, rather than our use case of as a build tool, then hopefully they'll submit an update to the package.